rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -2110,6 +2110,148 @@ private static Stream<BridgeConstants> provideBridgeConstants() {
         return Stream.of(BridgeRegTestConstants.getInstance(), BridgeTestNetConstants.getInstance(), BridgeMainNetConstants.getInstance());
     }
 
+    @Test
+    void rskTxWaitingForSignature_fail_adding_an_already_existence_key_after_rskip_375() throws IOException {
+        // Arrange
+        BridgeConstants bridgeConstants = BridgeRegTestConstants.getInstance();
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP176)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP375)).thenReturn(true);
```

Comment:
Can't you use Fingerrot activation directly?

Reply:
Done